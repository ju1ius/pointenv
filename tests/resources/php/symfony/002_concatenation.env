A='a''b'
B='a '' b'
C='a'\''b'
D="a""b"
E=a'b'"c"d
