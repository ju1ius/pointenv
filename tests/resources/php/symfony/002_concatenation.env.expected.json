{
    "A": "ab",
    "B": "a  b",
    "C": "a'b",
    "D": "ab",
    "E": "abcd"
}